src/evmTest/resources/solidity/MutualRecursion.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.4.9;
+
+contract MutualRecursion {
+
+    function isEven(uint n) returns (bool) {
+        if (n == 0)
+            return true;
+        else
+            return isOdd(n - 1);
+    }
+
+    function isOdd(uint n) returns (bool) {
+        if (n == 0)
+            return false;
+        else
+            return isEven(n - 1);
```

Comment:
I have no reason to doubt it would work. Mutual recursion within a single contract will simply be a series of jumps, and as for testing for StackOverflow standard recursion would suffice.

I believe that a more interesting test would mutual recursion between 2 different contracts. One of the aspects would be checking for call-depth overflow (though possibly that could be checked with a single contract calling itself)

But I'm just making notes here. This contract is simple enough so no need to change it, and we can add the test with 2 different contracts later.